composer.json

```diff
@@ -16,19 +16,19 @@
         "doctrine/doctrine-bundle": "^1.8",
         "doctrine/orm": "^2.6",
         "league/oauth2-server": "^7.2",
-        "sensio/framework-extra-bundle": "^3.0.0|^4.0.0|^5.0.0",
-        "symfony/framework-bundle": "~3.4|~4.0",
-        "symfony/psr-http-message-bridge": "^1.0",
-        "symfony/security-bundle": "~3.4|~4.0",
-        "zendframework/zend-diactoros": "^1.7|^2.1"
+        "nyholm/psr7": "^1.1",
```

Comment:
I think `nyholm/psr7` should be moved to `require-dev` and `psr/http-factory` should explicitly required here

Reply:
@ajgarlag Yeah, it didn't make sense to force the user to use a specific PSR-7/17 implementation. The `psr/http-factory` is now explicitly required and I've added instructions on how to switch between different implementations.